ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,742 @@
+appropriate care so to respect confidentiality and the applicable regulations.
+
+The ledger operator is advised to use the tools available to them to not divulge
+private user data, including those documented by PostgreSQL, to protect data at
+rest and using a secure communication channel between the Ledger API server and
+the PostgreSQL server.
+
+Ledger API over TLS
+===================
+
+To protect data on-the-wire and allow using the Ledger API over untrusted networks,
+the DAML-on-SQL Ledger API server implementation leverages gRPC's built-in TLS support
+to allow clients to verify the server's identity and encrypt the communication channel
+over which the Ledger API requests and responses are sent.
+
+To enable TLS, you need to specify the private key for your server and the certificate
+chain via ``java -jar daml-on-sql-<version>.jar --pem server.pem --crt server.crt``.
+By default, DAML-on-SQL requires client authentication as well. You can set a custom root
+CA certificate used to validate client certificates via ``--cacrt ca.crt``. You can
+change the client authentication mode via ``--client-auth none`` which will disable it
+completely, ``--client-auth optional`` which makes it optional or specify the default
+explicitly via ``--client-auth require``.
+
+Ledger API Authentication
+=========================
+
+By default, DAML-on-SQL does not use any authentication and accepts all valid Ledger API requests.
+
+To start DAML-on-SQL with authentication based on `JWT <https://jwt.io/>`__ tokens,
+use one of the following command line options:
+
+- ``--auth-jwt-rs256-crt=<filename>``.
+  \DAML-on-SQL will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public key loaded from the given X.509 certificate file.
+  Both PEM-encoded certificates (text files starting with ``-----BEGIN CERTIFICATE-----``)
+  and DER-encoded certificates (binary files) are supported.
+
+- ``--auth-jwt-es256-crt=<filename>``.
+  DAML-on-SQL will expect all tokens to be signed with ES256 (ECDSA using P-256 and SHA-256) with the public key loaded from the given X.509 certificate file.
+  Both PEM-encoded certificates (text files starting with ``-----BEGIN CERTIFICATE-----``)
+  and DER-encoded certicates (binary files) are supported.
+
+- ``--auth-jwt-es512-crt=<filename>``.
+  DAML-on-SQL will expect all tokens to be signed with ES512 (ECDSA using P-521 and SHA-512)     with the public key loaded from the given X.509 certificate file.
+  Both PEM-encoded certificates (text files starting with ``-----BEGIN CERTIFICATE-----``)
+  and DER-encoded certificates (binary files) are supported.
+
+- ``--auth-jwt-rs256-jwks=<url>``.
+  DAML-on-SQL will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public key loaded from the given `JWKS <https://tools.ietf.org/html/rfc7517>`__ URL.
+
+.. warning::
+
+  For testing purposes only, the following options may also be used.
```

Comment:
There's one option. No need to get plural.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6850/commits/55f9b613c1f18b0d6d8b752f682ba23741599f89